src/webgpu/util/conversion.ts

```diff
@@ -458,7 +465,8 @@ export function bool(value: boolean): Scalar {
   // WGSL does not support using 'bool' types directly in storage / uniform
   // buffers, so instead we pack booleans in a u32, where 'false' is zero and
   // 'true' is any non-zero value.
-  return new Scalar(TypeBool, value, new Uint32Array([value ? 1 : 0]));
+  const arr = new Uint32Array([value ? 1 : 0]);
+  return new Scalar(TypeBool, arr[0], arr);
```

Comment:
Sorry, this one really should use `value`, as this is no longer a boolean.

Reply:
Done